Libraries/SDHC/ff13/Source/ffconf.h

```diff
@@ -171,6 +174,8 @@
 
 #define FF_MIN_SS		512
 #define FF_MAX_SS		512
+//#define FF_MIN_SS		4096
```

Comment:
This needs a comment explaining in what situation the sector size of 4096 would be used.

We should also consider whether we want to wrap this in some variable that we have more control over from the build system.

Ex:
```C
#ifdef SOMEVARIABLE
#define FF_MIN_SS		512
#define FF_MAX_SS		512
#else
#define FF_MIN_SS		4096
#define FF_MAX_SS		4096
#endif
```